Pulling an existing image causes imagec to panic

imagec:

```
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0xf1de40, 0xc82000c080)
    /usr/local/go/src/runtime/panic.go:481 +0x3e6
main.CreateImageConfig(0xc82022e0a0, 0x0, 0x2, 0xc8204839a0, 0x0, 0x0)
    /opt/go/src/github.com/vmware/vic/cmd/imagec/imagec.go:387 +0xeda
main.main()
    /opt/go/src/github.com/vmware/vic/cmd/imagec/imagec.go:597 +0x1483
```

<!--
Thank you for contributing to VIC! Please review this checklist
before submitting your issue.

[ ] Search for duplicates before logging new issues
    https://github.com/vmware/vic/issues?q=is%3Aopen+is%3Aissue
[ ] Questions are best asked and answered at Slack
    https://vmware.slack.com/messages/wolverines/

For bug reports, please include the information below.
__________________________________________________________ -->

**VIC Version:**

master


**Expected behavior:**
```
Using default tag: latest
latest: Pulling from library/busybox
385e281300cc: Already exists
a3ed95caeb02: Already exists
Status: Image is up to date for library/busybox:latest
```
**Actual behavior:**
```
Using default tag: latest
latest: Pulling from library/busybox
385e281300cc: Already exists
a3ed95caeb02: Already exists
invalid character 'p' looking for beginning of value
```

--- a/cmd/imagec/imagec.go
+++ b/cmd/imagec/imagec.go
@@ -385,6 +385,10 @@ func WriteImageBlobs(images []*ImageWithMeta) error {
 // CreateImageConfig constructs the image metadata from layers that compose the image
 func CreateImageConfig(images []*ImageWithMeta, manifest *Manifest) error {
 
+	if len(images) == 0 {
+		return nil
+	}
+
 	imageLayer := images[0] // the layer that represents the actual image
 	image := docker.V1Image{}
 	rootFS := docker.NewRootFS()

